### DISCUSSION TITLE
1.2 Beta 3 Filling up Macintosh HD/cores with files

### DISCUSSION THREAD

**Discussion by dmuckala** [apohl79/audiogridder#543] [Help]
Hi, on a Macbook Pro 16" running Catalina/Logic--every time I have a server crash I'm getting HUGE core dump files...today i ran out of disk space (created almost 200GB of files) until I realized what was going on.  Is there a way to prevent this from happening?  I obviously can't risk this happening again.  Great job on the latest update by the way...truly amazing.

**Comment by CaBuffalo** [comment]
AG server only keeps some latest log / dump files, shouldn't keep piling up
anymore. At least, this is how it works on windows server.

How many total dump files do you get? How far does it go back?

On Sat, May 8, 2021 at 5:53 PM dmuckala ***@***.***> wrote:

> Hi, on a Macbook Pro 16" running Catalina/Logic--every time I have a
> server crash I'm getting HUGE core dump files...today i ran out of disk
> space (created almost 200GB of files) until I realized what was going on.
> Is there a way to prevent this from happening? I obviously can't risk this
> happening again. Great job on the latest update by the way...truly amazing.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/543>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FDIQTT5RP77F7Q5QLTMXMHVANCNFSM44OG34OQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by dmuckala** [reply]
They all happened in a span of 15 minutes or so.  It was while AG was scanning a bunch of plugins and kept crashing, probably in relation to a plugin it didn't like.  Obviously if that hadn't happened, then the .core files wouldnt have happened either but it's a bad outcome nonetheless if an incompatible plugin is found and it happens again.  This is on the latest version of AG by the way.

**Reply by dmuckala** [reply]
I deleted them but IIRC it was about 30 files ranging from around 3-8GB each.

**Reply by apohl79** [reply]
I will add an option to turn off core dumps. The cleanup does not look at the scan files.

**Reply by dmuckala** [reply]
Incredible!  thankyou!
